src/dune_rules/expander.mli

```diff
@@ -95,7 +95,11 @@ val expand_pform : t -> Value.t list Action_builder.t String_with_vars.expander
 module Static : sig
   val expand : t -> mode:'a String_with_vars.Mode.t -> String_with_vars.t -> 'a
 
-  val expand_path : t -> String_with_vars.t -> Path.t
+  (** [expand_path ?relative_to expander str] expands the string [str] and
+      interprets it as a path relative to the [expander]'s directory. If
+      specified, [relative_to] will act as the base path instead of the
+      [expander]'s directory *)
+  val expand_path : ?relative_to:Path.t -> t -> String_with_vars.t -> Path.t
```

Comment:
Instead of adding this argument, is it possible to just use `expand_str` and then interpret that relative to your path?

Reply:
Done